libs/auth/src/cryptography.ts

```diff
@@ -187,6 +187,8 @@ export function extractPublicKeyFromCert(
   return openssl(['x509', '-noout', '-pubkey', '-in', cert]);
 }
 
+const VERIFICATION_TIME_MARGIN_MINUTES = 10;
```

Comment:
10 mins of wiggle room felt about right to me. Any more and you probably do want a reminder to check your clocks.